### DISCUSSION TITLE
Excerpts from James Saunders' with paper for neoscore Formalization

### DISCUSSION THREAD

**Discussion by jgarte** [DigiScore/neoscore#44] [Ideas]
wdyt

https://www.james-saunders.com/with-paper-20068-2009/

![jswithpaper](https://user-images.githubusercontent.com/47760695/175179379-2d4d1d68-b139-4fc8-9fac-6d4e74cf600e.png)

**Comment by ajyoon** [comment]
Everything here would be pretty straightforward to implement. If we wanted to use some of these as example scores we'd need to get permission first for copyright reasons. Is this something you're interested in pursuing?

**Reply by jgarte** [reply]
@ajyoon 

Hi, sorry for the late reply but I don't think I'll be able to work on this any time soon. I just wanted to share the idea in case anybody else is willing to pick it up an implement it.
